src/useValidation.js

```diff
@@ -90,10 +90,15 @@ export const useValidation = (defaultValue, validationFn, config) => {
 
     return {
         value,
-        error,
         onChange,
         onBlur,
+        error,
         validate,
-        reset
+        reset,
+        props: {
```

Comment:
What's the purpose of this `props` object?

Reply:
Described in feature request linked in PR description.